test/cmdlineTests/standard_file_not_found/err

```diff
@@ -1 +1 @@
-File not found: standard_file_not_found/input.sol
+File not found: input.sol
```

Comment:
Why have test args been changed?

Reply:
Previously every test would get `input.sol` as argument by default (unless there was another `input.*` file in the test directory). Now it only gets it if `input.sol` actually exists.

The problem is that this particular test is for a case where `input.sol` does not exist (it checks the error message). The solution was to simply pass the file name explicitly.